src/components/realm-selector/RealmSelector.tsx

```diff
@@ -126,18 +141,12 @@ export const RealmSelector = () => {
           onSearchButtonClick={() => onFilter()}
           className="keycloak__realm_selector__context_selector"
         >
-          {recentUsed.used.map((realm) => (
-            <ContextSelectorItem key={realm}>
-              <RealmText value={realm} /> <Label>{t("recent")}</Label>
+          {(filteredItems || all).map((item) => (
```

Comment:
Note that you can also use the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) here to expicitly handle `undefined` and `null` values instead of 'falsy' values.

```suggestion
          {(filteredItems ?? all).map((item) => (
```

Reply:
you suggested that somewhere else as well I don't see the value in it as this list of items can only be undefined or an array of strings

Reply:
Yeah, this is just a personal preference.